lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -615,7 +615,14 @@ module.exports = class MethodMappings {
 
         async clickElement(webElementOrId) {
           const element = this.getWebElement(webElementOrId);
-          await element.click();
+
+          const {browserName = '', platformName = ''} = this.settings.capabilities;
+
+          if (browserName.toLowerCase() === 'safari' && platformName.toLowerCase() === 'ios') {
```

Comment:
can it not be done by checking `isMobile`

Reply:
I think `isSafari` and `isIos` would be better to use here